src/VisualizationTests/HelixWatch3DViewModelTests.cs

```diff
@@ -1276,6 +1276,28 @@ public void InstancedLinesAreAddedToBackGroundPreviewForEachMatrix()
             Assert.AreEqual(6 * 6 * 6, BackgroundPreviewGeometry.TotalLineInstancesToRender());
 
         }
+        [Test]
+        [Category("Failure")]
```

Comment:
we can enable this after the LibG changes are merged, it will fail until then because the number of instances will be incorrect.